framework/src/modules/keys/keys_module.ts

```diff
@@ -38,4 +52,172 @@ export class KeysModule extends BaseModule {
 			numberOfSignatures: 0,
 		},
 	};
+
+	// eslint-disable-next-line class-methods-use-this
+	public async beforeTransactionApply({
+		stateStore,
+		transaction: {
```

Comment:
not important, but if it is destructuring everything, then probably it's better to use as `transaction`?

Reply:
Done.